projects/rero/ng-core/src/lib/record/record-statistics/record-statistics.component.html

```diff
@@ -0,0 +1,31 @@
+<!--
+  RERO angular core
+  Copyright (C) 2020 RERO
+ 
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+ 
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+ 
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container>
+    <h5 id="stats" class="mt-5">{{'Statistics' | translate}}</h5>
+    <hr class="mb-4 mt-0"/>
```

Comment:
I don't think `id` attribute is necessary on header tag. If you really need it, you could place it on the `ng-core-record-statistics` from "detail.component.html". My opinion is that you could remove it.

Same remarks for the "class" attribute. Use `mt-5` on parent `ng-core-record-statistics` tag (the margin is not really on the header, but for the component). Additionally it will be more generic because somehow could not need "mt-5" is it uses this component in other page.

Next, place the `translate` pipe into the h5 tag.

Finally, it seems the next `hr` tag isn't necessary. What you need is a "border-bottom" on the H5 tag. Add a css-class with this declaration : `h5.full_underline { width: 100%; border-bottom: $border-width solid $border-color; }`

It will done : `<h5 class='full-underline' translate>`